src/libs/live_view.c

```diff
@@ -479,6 +479,24 @@ void view_enter(struct dt_lib_module_t *self,struct dt_view_t *old_view,struct d
   gtk_widget_set_sensitive(lib->focus_out_small, sensitive);
 }
 
+void view_leave(struct dt_lib_module_t *self, struct dt_view_t *old_view, struct dt_view_t *new_view)
+{
+  dt_lib_live_view_t *lib = self->data;
+
+  // there's no cdode to automatically restart live view when entering
+  // the view, and besides the user may not want to jump right back
+  // into leave view if they've been out of tethering view doing other
+  // things
+  if(gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(lib->live_view)) == TRUE)
+  {
+#if 0
```

Comment:
I have a fix for this one.

Reply:
That's great. I didn't look at this hard -- I just sketched out code and when it crashed left it off.

Reply:
Fixed in b3f830c43fce0abe4e0fcd14dcdf9c4f20c55020, you can reactivate this part.

Reply:
> Fixed in b3f830c, you can reactivate this part.

Just put up a commit to do that. I won't rebase this branch on top of b3f830c, to keep this PR cleaner.